.github/actions/smoke-test/test.sh

```diff
@@ -14,3 +14,11 @@ docker images
 
 # Clean up
 docker rm -f $(docker container ls -f "label=${id_label}" -q)
+
+# Checking size of universal image
+
+if [ $IMAGE == "test-universal" ]; then
```

Comment:
```suggestion
if [ $IMAGE == "universal" ]; then
```

See 👇 https://github.com/devcontainers/images/blob/8d583c121d54f913ea28b3790672b21c52df2a93/.github/workflows/smoke-universal.yaml#L28

Reply:
done